application/src/userlistmodel.cpp

```diff
@@ -0,0 +1,81 @@
+#include "userlistmodel.h"
+#include "databaseengine.h"
+
+UserListModel::UserListModel(QObject *parent) : QAbstractListModel(parent)
+{
+    connectDatabase();
+}
+
+void UserListModel::addUserlist(QList<User> userList)
+{
+
+    int m_size = m_UserList.size();
```

Comment:
```suggestion
    int size = m_UserList.size();
```
префиксы указываются только полям класса, чтобы их отличать от временных переменных